stytch/api/o_auth_service.py

```diff
@@ -0,0 +1,1547 @@
+# !!!
+# WARNING: This file is autogenerated
+# Only modify code within MANUAL() sections
+# or your changes may be overwritten later!
+# !!!
+
+from typing import Any, Dict, List, Optional, Union
+
+import pydantic
+
+from stytch.core.api_base import ApiBase
+from stytch.core.http.client import AsyncClient, SyncClient
+from stytch.models.o_auth_service import (
+  OauthattachResponse,
+  OauthgoogleonetapstartResponse,
+  OauthgooglestartResponse,
+  OauthgoogleidtokenauthenticateResponse,
+  OauthmicrosoftstartResponse,
+  OauthapplestartResponse,
+  OauthappleidtokenauthenticateResponse,
+  OauthgithubstartResponse,
+  OauthfacebookstartResponse,
+  OauthamazonstartResponse,
+  OauthbitbucketstartResponse,
+  OauthcoinbasestartResponse,
+  OauthdiscordstartResponse,
+  OauthfigmastartResponse,
+  OauthgitlabstartResponse,
+  OauthinstagramstartResponse,
+  OauthlinkedinstartResponse,
+  OauthshopifystartResponse,
+  OauthslackstartResponse,
+  OauthsnapchatstartResponse,
+  OauthspotifystartResponse,
+  OauthsteamstartResponse,
+  OauthtiktokstartResponse,
+  OauthtwitchstartResponse,
+  OauthtwitterstartResponse,
+  OauthauthenticateResponse,
+)
+
+
+class OAuthService:
+    def __init__(
+      self,
+      api_base: ApiBase,
+      sync_client: SyncClient,
+      async_client: AsyncClient,
+    ) -> None:
+        self.api_base = api_base
+        self.sync_client = sync_client
+        self.async_client = async_client
+
+    @property
+    def sub_url(self) -> str:
+        return "o_auth_service"
```

Comment:
looks like there's an unsafe assumption about suburls that I'm generating noncompliant input for